src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/edit_product_row.html.twig

```diff
@@ -37,6 +37,11 @@
   <td class="editProductRefunded"></td>
   <td class="editProductAvailable"></td>
   <td class="editProductTotalPrice"></td>
+  {% if orderForViewing.hasInvoice() %}
```

Comment:
Should you add the whole td or only the content? (It's not that important for now since the number of columns is not consistent in add/edit rows anyway But we will have to clear this before the freeze

Reply:
Well this column displays the invoice reference, but if there is no invoice... I thought it would be better to not display the column at all...